Bottom: 949635f7c24027618e63c2aeb0dfbb87ef2cc520
Top:    2168e13b16fc4d70d3c272d14b0ebdbf5f9a5b6e
Author: Jason Wang <jasowang@redhat.com>
Date:   2016-12-22 17:56:53 +0800

Refresh of virtio-net-forbid-xdp-when

---

diff --git a/drivers/net/virtio_net.c b/drivers/net/virtio_net.c
index c316525..7416349 100644
--- a/drivers/net/virtio_net.c
+++ b/drivers/net/virtio_net.c
@@ -1679,7 +1679,7 @@ static int virtnet_xdp_set(struct net_device *dev, struct bpf_prog *prog)
 
 	if (virtio_has_feature(vi->vdev, VIRTIO_NET_F_GUEST_TSO4) ||
 	    virtio_has_feature(vi->vdev, VIRTIO_NET_F_GUEST_TSO6) ||
-	    virtio_has_feature(vi->vdev, VIRTIO_NET_F_GUEST_UFO))) {
+	    virtio_has_feature(vi->vdev, VIRTIO_NET_F_GUEST_UFO)) {
 		netdev_warn(dev, "can't set XDP while host is implementing LRO, disable LRO first\n");
 		return -EOPNOTSUPP;
 	}
